pkgs/by-name/te/testlib/package.nix

```
@@ -0,0 +1,39 @@
+{
+  fetchurl,
+  stdenv,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "testlib";
+  version = "0.9.41";
+
+  src = fetchurl {
+    url = "https://github.com/MikeMirzayanov/testlib/releases/download/${finalAttrs.version}/testlib.h";
+    hash = "sha256-cPdMVw8rRdYwhq5uTEG7tcX/1EKMupkUu8A5bSm+ENg=";
+  };
+
+  outputs = [
+    "dev"
+    "out"
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out
+    mkdir -p $dev/include/testlib
+    cp ${finalAttrs.src} $dev/include/testlib/testlib.h
+    runHook postInstall
+  '';
+
+  dontUnpack = true;
+
+  meta = {
+    description = "This project contains a C++ implementation of testlib.";
```

```suggestion
    description = "This project contains a C++ implementation of testlib";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes